_docs/portal/1.200.0.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: 1.200.0
+permalink: /docs/portal/1.200.0/
+parent: Portal
+nav_order: -10001
+release_date: 15.07.2024
+---
+
+## Import users accepts e-mail addresses with capital letters
+Import users from .csv file should now accept e-mail addresses with capital letters. Previously, such users were filtered out.
```

Comment:
accepts e-mail addresses with capital letters. We don't need to know that from **now** on - because it's obvious that from a specific release